src/spec.c

```diff
@@ -1072,7 +1068,16 @@ FieldSpec *IndexSpec_CreateField(IndexSpec *sp, const char *name, const char *pa
   fs->ftWeight = 1.0;
   fs->sortIdx = -1;
   fs->tagFlags = TAG_FIELD_DEFAULT_FLAGS;
-  fs->tagFlags = TAG_FIELD_DEFAULT_SEP;
```

Comment:
This was a bug as `tagFlags` is shown twice. To cover for it, the fix was lines 323-326 which were removed once this is fixed.